src/app/components/TextInput.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from "../stylesheet/components/TextInput.module.css";
+export default function TextInput(props) {
+  return (
+    <input
+      type={props.type}
+      id={props.id}
+      name={props.name}
+      placeholder={props.placeholder}
+      className={styles.input}
+    />
+  );
+}
```

Comment:
Same as the previous `Label` component. If you are not doing something different/special, there is no need to create a component.